torch/csrc/lazy/python/init.cpp

```diff
@@ -1,12 +1,65 @@
 #include <torch/csrc/lazy/python/init.h>
 
+#include <c10/core/Device.h>
+#include <torch/csrc/jit/python/pybind.h>
+#include <torch/csrc/lazy/backend/backend_device.h>
 #include <torch/csrc/lazy/core/debug_util.h>
+#include <torch/csrc/lazy/core/lazy_graph_executor.h>
+#include <torch/csrc/lazy/core/metrics.h>
 #include <torch/csrc/lazy/python/python_util.h>
+#include <torch/csrc/lazy/backend/backend_interface.h>
+#include <torch/csrc/lazy/ts_backend/ts_backend_impl.h>
+#include <string>
+#include <vector>
 
 namespace torch {
 namespace lazy {
 
-void initLazyBindings(PyObject* /* module */){
+// TODO(whc) backend 'device' related APIs are not very clear, this code could be
+// simplified but it should probably be done together with designing/refactoring
+// the overall approach to get/set of default eager/lazy device types
+torch::lazy::BackendDevice GetDeviceOrCurrent(const std::string& device_str) {
+  if (device_str.empty()) {
+    getBackend()->GetDefaultDeviceType();
+    return torch::lazy::BackendDevice();
+  }
+  return torch::lazy::atenDeviceToBackendDevice(c10::Device(device_str));
+}
+
+void initLazyBindings(PyObject* module){
+  auto m = py::handle(module).cast<py::module>();
+  auto lazy = m.def_submodule("_lazy");
+  auto lazy_ts_backend = m.def_submodule("_lazy_ts_backend");
```

Comment:
In fact, this might be what you intended all along based on its usage in `test_op_info.py`:
https://github.com/pytorch/pytorch/pull/74508/files#diff-222fb8e63ff517f76308939550ca744de90dd6ea5706e2afd3f72db019df9b90R24

Reply:
yea its a good question;
I tried to make it clean from the actual user API side, which is how it looks when called from test_ts_opinfo.py.

generally the c++ bindings are flatter all over pytorch and then structurized at the glue-layer where .py files re-export them in a proper namespace (the stuff happening in torch/_lazy/*.py), so I didn't think it mattered too much.  

I might be convinced to change it though.  Wdyt @Krovatkin ?  One annoying thing is I would have to then define a 'submodule' correctly in the .pyi file that provides the mypy formatting, but i'm sure that's  not too hard